ImplementationGuide/markdown/Uebersicht.md

```diff
@@ -34,26 +34,20 @@ Ziel des _Smart App Launch_ ist es, ein Zugriffstoken von einem OAuth2-kompatibl
 
-Diese in ISiK Stufe 3 vorgenommene Profilierung des SMART-on-FHIR-API fokussiert somit auf die Absicherung der bereits in ISiK Stufe 2 betrachteten FHIR-Ressourcen wie z. B. _Patient_, _Encounter_ oder _Observation_, macht den Herstellern und Krankenhäusern hierbei aber weder Vorgaben zu konkreten Berechtigungsmodellen/-regeln oder dem Zuschnitt der zum Identitäts- und Berechtigungsmanagement (IAM) einzusetzenden Produkte. Hersteller von bereits in ISiK Stufe 2 bestätigungsrelevanten Systemen werden durch die vorliegende Spezifikation in die Lage versetzt werden, eine konforme Erweiterung der bestehenden Implementierung zu erstellen und das Bestätigungsverfahren der gematik für ISiK-Connect in ISiK Stufe 3 erfolgreich zu absolvieren, ohne dass hierzu tiefgreifende systemische Änderungen an der bestehenden Autorisierungs- und Authentifizierungsinfrastruktur erforderlich sind.
-
-## Informative Hinweise zur weitergehenden Umsetzung von SMART on FHIR
-
-Für Hersteller von Autorisierungsservern, _SMART Client-Apps_, KIS sowie Patienten- und Zuweiserportalen kann es attraktiv sein, ihre Lösungen über die in ISiK Stufe 3 normativen API-Aufrufe gegenüber ISiK-Ressourcenservern hinaus am SMART-on-FHIR-Standard auszurichten. Auch Krankenhäuser können von den grundlegenden Ideen hinter _SMART on FHIR_ profitieren, z. B. indem funktionale Module von Drittherstellern über einen _Smart App Launch_ einfacher in bestehende Systemlandschaften integriert werden können.
-
-ISiK Connect in der Stufe 3 unterstützt dieses, indem die Integration der bestätigungsrelevanten Systeme und Schnittstellen in das gesamte SMART-on-FHIR-Ökosystem als informative Ergänzung zu ISiK 3 beschrieben wird. Im Abschnitt ["SMART on FHIR"](ISiKundSMART.md) werden die kompletten Interaktionen zwischen EHR (KIS, Portal), _SMART Apps_, Autorisierungsserver und Ressourcenservern auf Basis der aktuellen Fassung des SMART-on-FHIR-Standards beschrieben. Diese Beschreibungen sind informativ, da sie aktuell nicht Gegenstand der ISiK-Konformitätsfeststellungen sind und da sie - zumindest in Teilen - auch über den Standard hinausgehende Good Pracitices beinhalten. Um den Lesern dieser Spezifikation die Unterscheidung zwischen normativen und informativen Bestandteilen von ISiK Connect zu erleichtern, sind alle Seiten mit informativen Inhalten deutlich mit dem Label "Informativ" im Seitenkopf gekennzeichnet.
+Diese in ISiK vorgenommene Profilierung des SMART-on-FHIR-API fokussiert auf die Absicherung der in anderen ISik-Modulen spezifizierten FHIR-Ressourcen wie z. B. _Patient_, _Encounter_ oder _Observation_. Hierbei macht die Spezifikation den Herstellern und Krankenhäusern hierbei aber weder Vorgaben zu konkreten Berechtigungsmodellen/-regeln oder dem Zuschnitt der zum Identitäts- und Berechtigungsmanagement (IAM) einzusetzenden Produkte.
```

Comment:
question: alle Beispiele in der Spec fokussieren sich auf die Basisressourcen (Patient, Encounter, Observation). Ist die Spec im jetzigen zustand eigentlich auch auf andere Fachmodule anwendbar (bspw. Terminplanung wo der Kontext weder Patient noch Encounter sein kann)?